istioctl/cmd/auth.go

```diff
@@ -102,41 +103,52 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 		},
 	}
 
-	upgradeCmd = &cobra.Command{
-		Hidden: true,
-		Use:    "upgrade",
-		Short:  "Upgrade Istio Authorization Policy from version v1alpha1 to v1beta1",
-		Long: `Upgrade converts Istio authorization policy from version v1alpha1 to v1beta1. It requires access to Kubernetes
-service definition in order to translate the service name specified in the ServiceRole to the corresponding
-workload labels in the AuthorizationPolicy. The service definition could be provided either from the current
-Kubernetes cluster or from a YAML file specified from command line.
+	convertCmd = &cobra.Command{
+		Use:   "convert",
+		Short: "Convert v1alpha1 RBAC policy to v1beta1 authorization policy",
+		Long: `Convert converts Istio v1alpha1 RBAC policy to v1beta1 authorization policy. The command talks to Kubernetes
+API server to get all the information needed to complete the conversion, including the currently applied v1alpha1
+RBAC policies, the Istio config-map for root namespace configuration and the k8s Service translating the
+service name to workload selector.
+
+The tool could also be used in offline mode without talking to the Kubernetes API server. In this mode,
+all needed information are provided through command line.
+
+Note: The converter tool makes best effort to keep the syntax unchanged when
+converting v1alph1 RBAC policy to v1beta1 policy. However, in some cases, strict
+mapping with equivalent syntax is not possible (e.g., constraints no longer valid
+in the new workload oriented model, converting a service name containing a wildcard
+to workload selector).
+
+We highly recommend you to review the converted policies before applying them.
```

Comment:
If its highly recommended we should probably just say something like "always review converted policies before applying them".